.changeset/fast-moments-mix.md

```diff
@@ -0,0 +1,12 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": tree
+---
+Fix typing derived from unions of `AllowedTypes` arrays
+
+Unions of array types provided as an [AllowedTypes](https://fluidframework.com/search/?q=AllowedTypes) used to result in incorrectly computed insertable content types.
```

Comment:
Do we really want to link to search results here? Or was the intention to link to a specific page?

Reply:
```suggestion
Unions of array types provided as an [AllowedTypes](https://fluidframework.com/docs/api/fluid-framework/allowedtypes-typealias) used to result in incorrectly computed insertable content types.
```